src/devices/vdml_rotation.cpp

```diff
@@ -16,8 +16,12 @@
 namespace pros {
 inline namespace v5 {
     
-Rotation::Rotation(const std::uint8_t port, const bool reverse_flag) : Device(port, DeviceType::rotation) {
-	pros::c::rotation_set_reversed(port, reverse_flag);
+Rotation::Rotation(const std::int8_t port, const bool reverse_flag) : Device(port, DeviceType::rotation) {
```

Comment:
This would be instead of the reverse flag.
So you should remove the reverse flag and just use the negative port